typescript/scripts/README.adoc

```diff
@@ -0,0 +1,38 @@
+:toc: macro
+
+= tBTC v2 Refund script
+
+This script explains what to do in case you need to refund your BTC.
+
+toc::[]
+
+== Refund script
+
+=== Prerequisites
+
+Please make sure you have the following prerequisites installed on your machine:
+
+- https://nodejs.org[Node.js] >14.18.0
+- https://yarnpkg.com[Yarn] >1.22.19
+
+tBTCv2 system prerequisite is that you can refund your BTC only after `refundLocktime` expires, which is currently set to 9 month.
```

Comment:
```suggestion
tBTCv2 system prerequisite is that you can refund your BTC only if it was not used for minting tBTC and after `refundLocktime` has expired.
In practice, you should wait about an hour longer than `refundLocktime` since the transaction is only allowed to enter the Bitcoin blockchain if the median time of the last 11 blocks is equal to or grater than `refundLocktime` 
```

Reply:
Yes, I like it, especially the info about "tBTC minting"